samples/drivers/jesd216/boards/nrf54h20dk_nrf54h20_cpuapp.conf

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SPI=n
+CONFIG_SPI_NOR=n
```

Comment:
hmm I think SPI should actually not be selected at all, since now drivers all select their deps

Reply:
Addressed in #82599.